react/hooks/useScrollDirection.js

```diff
@@ -8,16 +8,14 @@ import useScroll from './useScroll'
 const useScrollDirection = () => {
   const { scroll } = useScroll()
   const [scrollingUp, setScrollingUp] = useState(false)
-  const [scrollingDown, setScrollingDown] = useState(false)
   const [lastScroll, setLastScroll] = useState(0)
 
-  useLayoutEffect(() => {
+  useEffect(() => {
     setLastScroll(scroll)
-    setScrollingUp(lastScroll > scroll)
-    setScrollingDown(lastScroll < scroll)
+    scroll >= 1 && setScrollingUp(lastScroll > scroll)
```

Comment:
I don't think this works as a threshold—it'd be better to remove this and work on a threshold on a future PR